docs/platforms/javascript/guides/react/getting-started.mdx

```diff
@@ -0,0 +1,336 @@
+---
+title: "Getting Started"
+sidebar_order: 1
+description: "Learn how to get started with the extended functionality of the React SDK and Sentry."
+---
+
+<PlatformContent includePath="llm-rules-platform" />
+
+This guide covers the full getting started pocess for the React SDK and assumes you want to utilize Error Monitoring, Logs, Tracing and Spans, and Replays. For the basic quickstart, the [React Quickstart](/platforms/javascript/guides/react/) guide is a better starting point.
+
+<PlatformContent includePath="getting-started-prerequisites" />
```

Comment:
Should the prerequisites maybe be step 1? Because for the wizard, you already need the Sentry Account.